### PR TITLE
3 add serverless

### PR DESCRIPTION
**Pull request by sadlerw**
* remove JSON stringify and format on body html
* add serverless local instructions to readme
* add headers for swagger to serverless
* remove snyk - not needed
* change tests to mocha
* remove unneeded stuff

### CODE REVIEW

**Review by ericsorenson — APPROVED** [review]
Looks 🆗 to me

File: serverless-api/src/hello.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/**
+ * Hello World Simple Lambda
+ *
+ * @param event
+ * @param context
+ * @param callback
+ */
+
+exports.handler = (event, context, callback) => {
```
Comment (kkoskelin) [inline serverless-api/src/hello.js:9]: Can we use async/await patterns in lambda rather than callbacks?

File: web-client/.eslintrc.js [hunk 2]
```diff
@@ -28,8 +28,9 @@ module.exports = {
     },
   },
   env: {
-    es6: true,
+    'cypress/globals': true,
     browser: true,
+    es6: true,
```
Comment (kkoskelin) [inline web-client/.eslintrc.js:33]: 👊 for sorting :)

**Review by kkoskelin — APPROVED** [review]
Would like to eschew callback-based code wherever possible, but this is def. a good first-pass.

File: serverless-api/src/hello.js (hunk 1 above)
Comment (sadlerw) [reply serverless-api/src/hello.js:9]: lambda only supports callback here afaik but if you can find an example let's do it

**Pull request merged**